### ISSUE TITLE
[Guide] How to translate

### ISSUE THREAD

**Issue by smarroufin** [nuxt/website-v2#1777]
Hi guys,

Here is a guide to help you contribute to translations. We will try to keep it updated so stay tuned to this issue!

### :warning: New Language Integration :warning:
For any new language, one that does not exist yet in the `content` folder, we will require a Lead Translator in order to accept the initial Pull Request.
A Lead Translator is someone that will actively review any content related to its language (and can of course still contribute). This way we can refer to someone mastering its language and being the guarantor of a proper translation.
If you want to contribute and make this initial Pull Request but do not want to be Lead Translator, please refer to someone that can take care of this.

### TL;DR
- `content/{lang}`: `en` is the reference folder so please refer to it to make sure you are synced with it.
- `i18n/{locale}.js`: `en-US.js` is the reference file.

### `content` folder
This is where all the markdown files are, which are processed by Docus to generate all those beautiful pages.
If you want to improve an existing language, jump into the concerned content sub-folder, make a pull request of your translation/improvement and that's it.
If the language does not exist, you will have to copy every file from the `content/en` folder to `content/{lang}` and start from there. Even if you don't translate some pages let them as they are, in english, because there is no fallback mechanism at the moment.
Don't worry about trying to translate every page at once, this can be made progressively.

### `i18n` folder
This is where you will find every translation injected in vue components.
Even if there is an `en` fallback, it is appreciated to keep this file as much synced as possible with `i18n/en-US.js`.

### Language availability
You can probably see many languages already translated into `content` or `i18n`. It does not mean it is available on the website. For the language to be available, we will need to add it to the `nuxt.config` > `i18n` configuration.
This will be taken care of by a repo maintainer / approved translator from the moment the language is translated enough.

### Contributors
We need the community contributions and we really appreciate all the work you guys are offering to us. You are welcome to make pull requests, even for a minor typo.
If you feel dedicated and want to actively contribute to translations, tell us, so you can be reviewer of other contributors' pull requests.

Thank you guys for all your past and future contributions, this means a lot to us :green_heart:
